.github/workflows/build_deploy.yml

```diff
@@ -0,0 +1,33 @@
+name: Build/deploy web pages
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+jobs:
+  build-and-maybe-deploy:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.6
+          bundler-cache: true
```

Comment:
I haven't looked at Ruby stuff in a long while. Does this use the repo's [Gemfile](https://github.com/CHTC/chtc-website-source/blob/master/Gemfile) and lock the Jekyll version by virtue of the locked Ruby version?

Reply:
I have no idea. This is basically the equivalent of the
```
rvm:
  - 2.6.3
```
block from travis.

Reply:
We should consider version locking Jekyll in the `Gemfile` then to be sure